### DISCUSSION TITLE
Start Menu application priority

### DISCUSSION THREAD

**Discussion by wards999** [Flow-Launcher/Flow.Launcher#1136] [Ideas]
Is there a way to force the first match in the results to that in the Start Menu?

For example, I have the epub reader Calibre installed - but when I type Cali intro Flow Launcher I get 

![cal](https://user-images.githubusercontent.com/83018031/164282423-d5794903-3da4-4546-9490-d24831c9072c.png)

and the 5th entry should be the one at the top

**Comment by taooceros** [comment]
By design, if you select the result you want a few time. It will be bumped up. It will be hard to say whether the priority of start menu may work because most programs are in it.

**Reply by wards999** [reply]
That's why I was thinking perhaps a custom plugin could do this - it could be specified as highest priority and only show things that are in the start menu?

I want to use Flow Launcher as a full start menu replacement - the good thing about the start menu is it only has 'key' applications displayed. Other EXEs that are installed but shouldn't normally be used are not listed.

**Comment by taooceros** [comment]
I think it is a reasonable assumption that apps in the start menu may be used more (as well as the directory specified by user).  Curious where do we get the other result.

**Comment by deefrawley** [comment]
You can also right click on the result you want and set as "Topmost for this query"

**Comment by wards999** [comment]
Thanks for this info @deefrawley, I didn't know about this feature.

However, my point remains that the entries in the start menu have generally been placed there because they are more used / useful - so it would be good to be able to have the option to specify that the values found in:

C:\ProgramData\Microsoft\Windows\Start Menu\Programs
C:\Users\<User>\AppData\Roaming\Microsoft\Windows\Start Menu\Programs

are the automatic 'first in the list' entries
